### PR TITLE
Split SDK file into multiple source files

### PR DESCRIPTION
**Pull request by yangyansong-adbe**
For more details, please refer to the task description. https://github.com/adobe/aepsdk-roku/issues/39

### CODE REVIEW

File: code/AdobeEdge.brs [hunk 1]
```diff
@@ -1,16 +1,18 @@
...
+' * of the License at http://www.apache.org/licenses/LICENSE-2.0
+' *
+' * Unless required by applicable law or agreed to in writing, software distributed under
+' * the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+' * OF ANY KIND, either express or implied. See the License for the specific language
+' * governing permissions and limitations under the License.
+' *
 ' *****************************************************************************************
 
+' *********************************** MODULE: pubic API ***********************************
```
Comment (addb) [inline code/AdobeEdge.brs:14]: typo: Public API

File: sample/development/README.md [hunk 2]
```diff
@@ -0,0 +1,24 @@
+# About
+
+This Roku channel is build for SDK development.
```
Comment (addb) [inline sample/development/README.md:3]: nit: built*

File: sample/development/components/SimpleVideoScene.brs [hunk 3]
```diff
@@ -0,0 +1,128 @@
...
+  ADB_CONSTANTS = AdobeSDKConstants()
+  m.adobeEdgeSdk.setLogLevel(ADB_CONSTANTS.LOG_LEVEL.VERBOSE)
+
+  ' get_mid_from_media_sdk = "12340203495818708"
+  ' m.adobeEdgeSdk.setExperienceCloudId(get_mid_from_media_sdk)
+
+  configuration = {}
+
+  test_config = ParseJson(ReadAsciiFile("pkg:/source/test_config.json"))
+  if test_config.count() > 0
```
Comment (addb) [inline sample/development/components/SimpleVideoScene.brs:29]: Add null/invalid check

File: code/main/common/stateManager.brs [hunk 4]
```diff
@@ -0,0 +1,169 @@
...
+        getConfigId: function() as dynamic
+            return m._edge_configId
+        end function,
+
+        getEdgeDomain: function() as dynamic
+            return m._edge_domain
+        end function,
+
+        updateECID: function(ecid as dynamic) as void
+            if ecid = invalid
```
Comment (addb) [inline code/main/common/stateManager.brs:70]: let's get changes from this PR https://github.com/adobe/aepsdk-roku/pull/40 before merging.

File: code/main/common/stateManager.brs [hunk 5]
```diff
@@ -153,7 +153,7 @@ function _adb_StateManager() as object
                     return invalid
                 end if
             else
-                _adb_log_error("_queryECID() - Error occured while quering ECID from service side. Please verify the edge configuration.")
+                _adb_log_error("_queryECID() - Error occured while quering ECID from service side. Please verify the edge configuration.The response code : " + StrI(response.code))
```
Comment (addb) [inline code/main/common/stateManager.brs:156]: nit: let's use `FormatJSON(response.code)`

File: sample/development/.vscode/launch.json [hunk 6]
```diff
@@ -8,7 +8,7 @@
             "host": "${config:roku_ip}",
             "password": "${config:roku_password}",
             "rootDir": "${workspaceFolder}",
-            // "injectRdbOnDeviceComponent": true,
+            "injectRdbOnDeviceComponent": true,
             // "deepLinkUrl": "http://${host}:8060/launch/dev?RunUnitTests=true"
         }
```
Comment (addb) [inline sample/development/.vscode/launch.json:13]: we need the curly bracket `}`

Comment (yangyansong-adbe) [reply sample/development/.vscode/launch.json:13]: Haha, we have the right `{}` pairs. It's a bug of Github to show those weird red lines here.

**Review by addb — APPROVED** [review]

**Pull request merged**